include/zephyr/ipc/ipc_static_vrings.h

```diff
@@ -25,8 +25,8 @@ extern "C" {
 /** Number of used VRING buffers. */
 #define VRING_COUNT	(2)
 
-/** VRING alignment. */
-#define VRING_ALIGNMENT	CONFIG_IPC_SERVICE_STATIC_VRINGS_ALIGNMENT
+/** Memory alignment. This should be aligned to the cache line. */
```

Comment:
Proposal:
If the cache is enable must be aligned on cache line, else should be aligned on processor 
architecture ( e.g 32-bits).

Reply:
I'll fix the comment.